app/models/indicadores_municipios_equipes_homologadas.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy import Column, Integer, String, Float, Numeric, DATE

```

Comment:
O tipo `Numeric` foi importado, mas não foi usado nesse model. O que acha de removê-lo das importações?
```suggestion
from sqlalchemy import Column, Integer, String, Float, DATE

```